### DISCUSSION TITLE
Axis NVR API

### DISCUSSION THREAD

**Discussion by alessandroFerraioliLeonardo** [AxisCommunications/discussions#4] [Axis Windows SDKs] [answered]
Dear all,

I'm Alessandro Ferraioli and I'm working on the integration of the AXIS NVR. I'm using the 2.17 version and according to the documentation, there is no _Update_ or _Remove_ for a camera. I was able to use the following endpoint:
```
URI: /Acs/Api/CameraFacade/AddCamera
```
in order to add a new camera. Unfortunately, I didn't find anything on the documentation regarding the _update_ or _remove_.

Can I get some feedback about that?

Best,

Alessandro

**Comment by mattias-kindborg-at-work** [comment, marked as answer]
Hi @alessandroFerraioliLeonardo!

This would be the ACS API I presume. Let me see if I can figure out whether updating or removing cameras is supported.

**Reply by alessandroFerraioliLeonardo** [reply]
Yep, it's the ACS API. Cheers mate, you're a lifesaver!

**Reply by mattias-kindborg-at-work** [reply]
Hi @alessandroFerraioliLeonardo.

Unfortunately the access to documentation and development support for AXIS Camera Station integration interfaces and tools are available to selected partners only. I think you need to contact your Axis account manager to initiate a discussion regarding this topic.
